ocular/tests/registry_cache.rs

```diff
@@ -0,0 +1,51 @@
+#![cfg(all(feature = "registry-cache"))]
+use assay::assay;
+use ocular::registry::{RegistryCache, Tag};
+
+#[assay]
+async fn registry_cache_happy_path() {
```

Comment:
Think it might be a good idea to get an unhappy path test here to verify things fail as expected.